### PR TITLE
Update ru.yml translations

### PR DESCRIPTION
**Pull request by MrSaturnBoing**

### CODE REVIEW

File: bot/data/locales/ru.yml [hunk 1]
```diff
@@ -49,19 +52,63 @@ misc:
...
+    invalid_server_id: "[ Неверный ID сервера. ]"
+    uncached_server: "-# не найдено имя сервера (меня в нём нет)"
+    get_server_id: "(Настройки Discord -> Расширенные -> включите \"Режим разработчика\", затем посмотрите на нижнюю часть контекстного меню сервера)"
+  
+  server:
+    welcome:
+      editor:
+        title: "[ Вы изменяете сообщение о приветствии... ]"
+        input: "Сообщение о приветствии"
+        placeholder: "Оставьте бланк пустым чтобы отключить функцию
```
Comment (fily-gif) [inline bot/data/locales/ru.yml]: Isn't it supposed to be "[...] чтобы отключить **эту** функцию"?

File: bot/data/locales/ru.yml [hunk 2]
```diff
@@ -564,11 +633,11 @@ nikogotchi:
         - "Немного подправить, а? Теперь я чувствую себя готовой к целому конвейеру!"
     penguin:
       fed:
-        - "HELLO."
+        - "ХЕЛЛО."
```
Comment (fily-gif) [inline bot/data/locales/ru.yml:636]: HELLO. are not meant to be translated

**Review by fily-gif — CHANGES_REQUESTED** [review]
Overall lgtm, just those changes and it'll be good to merge.

File: bot/data/locales/ru.yml (hunk 2 above)
Comment (meqativ) [reply bot/data/locales/ru.yml:636]: they're meant to be transliterated

Comment (fily-gif) [reply bot/data/locales/ru.yml:636]: Lgtm to me then

File: bot/data/locales/ru.yml [hunk 3]
```diff
@@ -49,19 +52,63 @@ misc:
...
+     
+# ---------------------------------------------------------------
+
+wool: ~
+
+# ---------------------------------------------------------------
+
+settings:
+  errors:
+    missing_permissions: "[ You must have the \"Manage Server\" permission in order to change the settings ]"
```
Comment (fily-gif) [inline bot/data/locales/ru.yml]: This is also meant to be translated btw

**Review by fily-gif — APPROVED** [review]
Looks Good To Me

**Pull request merged**